src/collection-view.js

```diff
@@ -482,7 +487,13 @@ Marionette.CollectionView = Marionette.View.extend({
 
   // render the child view
   renderChildView: function(view, index) {
+    if (!(view instanceof Marionette.View)) {
+      Marionette.triggerMethodOn(view, 'before:render', view);
+    }
     view.render();
+    if (!(view instanceof Marionette.View)) {
```

Comment:
It's an unfortunate guard, but the goal here is to target Backbone views for external `render` event triggering-on.